applications/serial_lte_modem/doc/MQTT_AT_commands.rst

```diff
@@ -0,0 +1,254 @@
+.. _SLM_AT_MQTT:
+
+MQTT AT Commands
+****************
+
+The following commands list contains the AT commands used to operate the MQTT client:
+
+* `MQTT Connect #XMQTTCON`_
+* `MQTT Subscribe #XMQTTSUB`_
+* `MQTT Publish #XMQTTPUB`_
+
+----
+
+MQTT Connect #XMQTTCON
+======================
+
+The ``#XMQTTCON`` command allows to connect to and disconnect from the MQTT broker.
+
+Set command
+-----------
+
+The set command allows to control to connect to and disconnect from the MQTT broker.
+
+Syntax
+~~~~~~
+
+::
+
+   AT#XMQTTCON=<op>[,<client_id>,<url>,<port>[,<sec_tag>]]
+
+
+* The ``<op>`` parameter is an integer and can accept one the following values:
+
+  * ``0`` - Connect to the MQTT broker.
+  * ``1`` - Disconnect from the MQTT broker.
+
+* The ``<client_id>`` parameter is a string and it indicates the MQTT Client ID.
+* The ``<url>`` parameter is a string and it indicates the MQTT broker hostname.
```

Comment:
* The ``<username>`` parameter is a string and it indicates the MQTT Client username.
* The ``<password>`` parameter is a string and it indicates the MQTT Client password.

Reply:
added